packages/rspack/tests/configCases/loader/builtin-loader-ident/webpack.config.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+	module: {
+		rules: [
+			{
+				use: [
+					{
+						loader: "builtin:swc-loader",
+						options: {
+							jsc: {
+								parser: {
+									syntax: "typescript"
+								}
+							}
+						},
+						ident: "builtin-swc-loader"
```

Comment:
I'm not sure. `ident: "builtin-swc-loader"` and `loader: "builtin:swc-loader"` are not intuitive. It's better that they could have the name.

Reply:
> It's better that they could have the name.

I can barely understand this. Would you elaborate?

Reply:
Ignore this. I misunderstand the feature.